client/blocks/login/social.jsx

```diff
@@ -44,6 +68,11 @@ class SocialLoginForm extends Component {
 						clientId={ config( 'google_oauth_client_id' ) }
 						responseHandler={ this.handleGoogleResponse } />
 				</div>
+
+				{ this.state.bearerToken &&
+					<WpcomLoginForm log={ this.state.username }
+									authorization={ 'Bearer ' + this.state.bearerToken }
+									redirectTo="/start" /> }
```

Comment:
I believe that's github using 8 characters for tabs! Try to load it in your IDE ;)